src/harness/templates/gateway.yaml

```diff
@@ -3,11 +3,11 @@
 apiVersion: networking.istio.io/v1beta1
 kind: Gateway
 metadata:
-  name: public
-  namespace: istio-system
+  name: {{ .Values.global.istio.gateway.name | default "public" }}
+  namespace: {{ .Values.global.istio.gateway.namespace | default "istio-system" }}
 spec:
   selector:
-     istio: ingressgateway
```

Comment:
usually selector is a list of key-value pairs. We can adhere to that so that if someone has any other selector, they can use it

Reply:
yes, fixed it. Can you review now?